.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -14,7 +14,12 @@ jobs:
   steps:
   - script: |
       # sudo pip install --upgrade pip
+      #sudo apt-get update
+      #sudo yum install mesa-libGL.x86_64
+      #sudo apt-get install -y libgl1-mesa-dev
+      #sudo apt-get install qtbase5-dev
       sudo pip install setuptools shyaml
+      #export DISPLAY=:0.0
```

Comment:
Please do not modify files outside of the recipes directory

Reply:
On it!

Reply:
Done.

Reply:
My apologies.  It was left over from troubleshooting the OpenGL issue on Linux.